crates/router/tests/connectors/bluesnap_ui.rs

```diff
@@ -0,0 +1,62 @@
+use serial_test::serial;
+use thirtyfour::{prelude::*, WebDriver};
+
+use crate::{selenium::*, tester};
+
+struct BluesnapSeleniumTest;
+
+impl SeleniumTest for BluesnapSeleniumTest {
+    fn get_connector_name(&self) -> String {
+        "bluesnap".to_string()
+    }
+}
+
+async fn should_make_3ds_payment(c: WebDriver) -> Result<(), WebDriverError> {
```

Comment:
Can we avoid single letter variable names?

Reply:
Done!